sdks/python/container/base_image_requirements.txt

```diff
@@ -67,7 +67,7 @@ pandas==0.23.4
 protorpc==0.11.1
 python-gflags==3.0.6
 setuptools<=39.1.0 # requirement for Tensorflow.
-tensorflow==1.11.0
+tensorflow==1.13.1
```

Comment:
`pyyaml` and `tensorflow` version upgrade is because of py3.7. Previous versions don't work for py3.7.

Reply:
Could you upgrade these dependencies to match (a) setup.py (latest in the range if possible) and (b) https://cloud.google.com/dataflow/docs/concepts/sdk-worker-dependencies

We should probably try to figure out a way to keep version in sync. This was recently discussed in the mailing list. For ideas you can look at that (https://lists.apache.org/thread.html/e174967bab6c224db40f66af74e7d07466812cc6b78189e55da88cb0@%3Cdev.beam.apache.org%3E)

Reply:
I compared with libraries installed by default at dataflow workers and matched versions to dataflow workers. For libraries whose versions are not in the range of `setup.py`, I updated version range accordingly.